pkg/store/bucket_e2e_test.go

```diff
@@ -134,10 +134,44 @@ func testBucketStore_e2e(t testing.TB, ctx context.Context, s *storeSuite) {
 		req      *storepb.SeriesRequest
 		expected [][]storepb.Label
 	}{
+		//{
```

Comment:
debug in progress - something does not work properly with `allWithout` function.